pkg/service/persistence.go

```diff
@@ -10,6 +12,10 @@ type Store interface {
 	Reset() error
 }
 
+type ExpiringData interface {
+	Expiration() time.Duration
```

Comment:
use TimeToLive() or TTL()?

Expiration() looks like it will return a specific time point?

Reply:
I just followed the naming of redis client. It uses "expiration" as `time.Duration`.